perf/ftdc_rollups.go

```diff
@@ -0,0 +1,207 @@
+package perf
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/evergreen-ci/sink/model"
+	"github.com/mongodb/ftdc"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultVer = 1
+)
+
+type performanceStatistics struct {
```

Comment:
can we merge this with the one in model?

Reply:
or maybe just use events?

Reply:
Once events.Performance has duration as cumulative we can just use that.

Reply:
let's open a ticket and do this in the next sprint.